website/templates/profile/notifications.mako

```diff
@@ -44,13 +45,23 @@
                     <!-- Flashed Messages -->
                     <div data-bind="html: message, attr: {class: messageClass}"></div>
             </div><!--view model scope ends -->
-            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"><h3 class="panel-title">Configure Notification Preferences</h3></div>
+                <form id="selectNotifications" class="osf-treebeard-minimal">
+                    <div id="grid"></div>
+                    <div class="help-block" style="padding-left: 15px">
+                            <p id="configureNotificationsMessage"></p>
+                    </div>
+                </form>
```

Comment:
Yeah, growl doesn't make sense here, although I respect Harry's desire for
some message. Let's do inline. Maybe Harry can hack that up.

On Thu, Feb 26, 2015 at 4:10 PM, Saman Ehsan <notifications@github.com>
wrote:

> In website/templates/profile/notifications.mako
> <https://github.com/CenterForOpenScience/osf.io/pull/1772#discussion_r25464625>
> :
>
> > @@ -44,13 +45,23 @@
> >                      <!-- Flashed Messages -->
> >                      <div data-bind="html: message, attr: {class: messageClass}"></div>
> >              </div><!--view model scope ends -->
> > -            </div>
> > +        </div>
> > +        <div class="panel panel-default">
> > +            <div class="panel-heading"><h3 class="panel-title">Configure Notification Preferences</h3></div>
> > +                <form id="selectNotifications" class="osf-treebeard-minimal">
> > +                    <div id="grid"></div>
> > +                    <div class="help-block" style="padding-left: 15px">
> > +                            <p id="configureNotificationsMessage"></p>
> > +                    </div>
> > +                </form>
>
> I think an inline message with treebeard might be the way to go then. If
> I'm not mistaken, the growl notifications stack up so if a user configures
> several notifications at once they would have to manually close them all.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/1772/files#r25464625>
> .
>